.gitignore

```diff
@@ -1,4 +1,6 @@
 .DS_Store
+yarn.lock
```

Comment:
@brooklynrob - please remove this entry, we want `yarn.lock` to be pushed into GitHub, see https://legacy.yarnpkg.com/blog/2016/11/24/lockfiles-for-all/

Reply:
Changed back and added as a new commit.